hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -835,8 +844,15 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 	if err != nil {
 		return ctrl.Result{}, fmt.Errorf("failed to hash HostedCluster configuration: %w", err)
 	}
+	additionalTrustBundle := ""
```

Comment:
Should this whole block be dropped? The same code is above at L632.

Reply:
Right, I mean just return the whole CM back as an object then just get the name and data when you need it from that object rather than retrieving it twice for different reasons.